### DISCUSSION TITLE
Not able to push the code to githib

### DISCUSSION THREAD

**Discussion by jbajaj19** [community/community#27297] [New to GitHub]
When I gave the command "git push origin master" in command prompt and hit enter then a window pops up asking for authenticating yourself either through browser or through code, when I click on browser and provide credentials, my account is authenticated but when I come back to command prompt then I am displayed with a message

fatal : An error occured while sending the request
fatal : The request was aborted : could not create SSL TSL secure channel.

Any suggestion will be appreciated.

**Comment by jfullstackdev** [comment]
to solve it, you can drag and drop on the Web UI for those changed files, 

or if that is a little bit complicated, download GitHub Desktop even without Git, you can still correctly push commits, 

in order to have the verified signature, my technique is to not push directly but through a pull request

**Comment by kurnakovv** [comment]
@jbajaj19 You can setup ssh key, and after that communication with github will be secure. You can use this manual:
https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/GitHub-SSH-Key-Setup-Config-Ubuntu-Linux
